[Feature] Character counter for TextInput and Textarea (#752)
vrk-kpa/suomifi-ui-components: src/core/Form/TextInput/TextInput.md

  />
</>;
```

### TextInput with character counter

```js
import { TextInput } from 'suomifi-ui-components';
import React, { useState } from 'react';

const [isError, setIsError] = useState(false);
const [statusText, setStatusText] = useState('');

const maxCharAmount = 20;

/**
 * Check if maximum amount of characters has exceed, and set status and statusText accordingly.

>  * You can also perform all your other validation needs here.

Reviewer: ```suggestion
 * You can also add other input validation to statusText.
```

Author: This would imply that validation is added **to** statusText itself and I find it more confusing than the old one. How about something like "You can also add any other desired input validation rules here."? The example itself shows the usage of statusText as it is.

Reviewer: Good point, that sounds good. 👍
